modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldSet/FieldSet.es.js

```diff
@@ -98,10 +102,18 @@ const FieldSet = ({
 						}
 						title={label}
 					>
-						{renderLayout(otherProps)}
+						<Layout
+							components={components}
+							editable={type === 'fieldset' && !ddmStructureId}
```

Comment:
I investigated and I didn't saw anything very specific for not use `editable` received from FieldSet component. See the commit https://github.com/liferay/liferay-portal/commit/5514bb479c6a133fbd559731243fc2750f3bd5a3

@albinojunior, Do you remember?

Reply:
Hey guys, I used this condition to avoid the creation of drag rows inside the fieldset when it is added to form builder during the form creation(bug fix). At the moment it was the only way that we found to prevent this. We can discuss more about it.

Reply:
@albinojunior I see! I think we can resolve it in another way but we can leave it for later. @diegonvs can you verify that with the changes to have an EditorVariant this problem is repeated?

Reply:
`editable` property that FieldSet component receives means the component is being rendered on the form builder view. However We must pass the given verification for disabling the `edit` mode on the FieldSet, if you take a look on Data Engine's FieldSet, We can't resize and hover on the FieldSet fields

Reply:
I'll maintain the `editable={type === 'fieldset' && !ddmStructureId}` :)